### DISCUSSION TITLE
Highlight Extension Slider Dots

### DISCUSSION THREAD

**Discussion by dheerajKsharma** [breezefront/community#168] [1. Help]
Hi,
How to enable slider dots for https://docs.swissuplabs.com/m2/extensions/highlight/ extension?

**Comment by dheerajKsharma** [comment]
Any help?

**Comment by vovayatsyuk** [comment]
Hi, I think this module doesn't have a "dots" option.

p.s. Please note, that these discussions are made to talk about breeze-related topics. Please post the questions about swissup modules to the siwssuplabs.com website using your account.
